CRUD.cs

```diff
@@ -266,4 +267,12 @@ public static void DeleteMovie()
             Helpers.DisplayMovieList();
         }
     }
-}
\ No newline at end of file
+}
+//// Continuously prompts the user for input until a valid (non-null and non-whitespace) movie title is provided.
+//while (string.IsNullOrWhiteSpace(movieTitle))
```

Comment:
this is a nice way to do input validation, as the user gets feedback about their incorrect inputs instead of the program crashing. the program will not proceed until the user enters a valid input, which helps with the flow of the program.